### PR TITLE
Small doc updates for installation

### PR DESCRIPTION
**Pull request by ru-fu**
Some updates to the "Install dependencies" and "Installing the doc processors" sections - small languages fixes plus adding IDs/comments that make it easier to link and reuse.

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]
All looks reasonable to me.

**Review by gmarull — APPROVED** [review]
LGTM

**Review by nashif — APPROVED** [review]

File: doc/README.rst [hunk 1]
```diff
@@ -64,6 +64,8 @@ described in :ref:`getting_started`. Then install additional tools
 that are only required to generate the documentation,
 as described below:
 
+.. doc_processors_installation_start
```
Comment (mbolivar-nordic) [inline doc/README.rst:67]: Should be `.. _doc_processors_installation_start:`

File: doc/README.rst [hunk 2]
```diff
@@ -113,10 +117,12 @@ as described below:
       .. note::
          On Windows, the Sphinx executable ``sphinx-build.exe`` is placed in
          the ``Scripts`` folder of your Python installation path.
-         Dependending on how you have installed Python, you may need to
+         Dependending on how you have installed Python, you might need to
          add this folder to your ``PATH`` environment variable. Follow
          the instructions in `Windows Python Path`_ to add those if needed.
 
+.. doc_processors_installation_end
```
Comment (mbolivar-nordic) [inline doc/README.rst:124]: `.. _doc_processors_installation_end:`

**Comment by mbolivar-nordic** [comment]
@ru-fu I'm assuming you didn't mean for those to be comments, did you?

**Comment by ru-fu** [comment]
> @ru-fu I'm assuming you didn't mean for those to be comments, did you?

Yes, I did. For reuse, we need any string (thus an ID or a comment). And in this case, I don't see any value in adding an ID, because if you want to link, you want to link to the full section (which already has an ID).

**Comment by mbolivar-nordic** [comment]
OK, I see. By "reuse" you meant with `include` + `start-after`.

**Pull request merged**